socorro/unittest/external/es/test_supersearch.py

```diff
@@ -561,6 +561,16 @@ def test_get_with_facets(self):
             _facets=['unkownfield']
         )
 
+    def test_get_missing_index(self):
```

Comment:
Minor nit, but "missing" is ambiguous here.  To be clear, this means "test what happens if a request for a non-existent index is made", correct?  If so, prefer something like `test_get_nonexistent_index`.  Again, minor, but to strive for clarity is a noble and virtuous pursuit. :haircut: